OpenRA.Mods.RA/ImportRedAlertLegacyMapCommand.cs

```diff
@@ -126,6 +126,17 @@ public override string ParseTreeActor(string input)
 			return input.ToLowerInvariant();
 		}
 
+		public override CPos ParseActorLocation(string input, int loc)
+		{
+			var newLoc = new CPos(loc % MapSize, loc / MapSize);
+			var vectorDown = new CVec(0, 1);
+
+			if (input == "tsla" || input == "agun" || input == "gap" || input == "apwr")
```

Comment:
OK, so it looks like this should just contain `iron` as well. No need to add it to the upgrade rules since its rules haven't changed.